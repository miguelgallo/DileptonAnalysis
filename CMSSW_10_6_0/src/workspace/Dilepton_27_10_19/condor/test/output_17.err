Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
ImportError: No module named site
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCand
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCandOutOfAccept
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> nGenProtCand
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_status
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_mass
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dpt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dphi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_3Dangle

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x00002afebb07541c in waitpid () from /lib64/libc.so.6
#1  0x00002afebaff2f12 in do_system () from /lib64/libc.so.6
#2  0x00002afeba94b52d in TUnixSystem::StackTrace() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#3  0x00002afeba94ddd4 in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#4  <signal handler called>
#5  0x00002afec7b63e5c in ?? ()
#6  0x3f915eb860000000 in ?? ()
#7  0xbf807851e0000000 in ?? ()
#8  0xbfce570a40000000 in ?? ()
#9  0x3fc18cccc0000000 in ?? ()
#10 0xbfe0b851e0000000 in ?? ()
#11 0xc006b33340000000 in ?? ()
#12 0xc01651eb80000000 in ?? ()
#13 0x3fb7a3d700000000 in ?? ()
#14 0xc034f0a3e0000000 in ?? ()
#15 0xbfdb75c280000000 in ?? ()
#16 0x3fb7b851e0000000 in ?? ()
#17 0x3f5775c280000000 in ?? ()
#18 0xc01d70a3e0000000 in ?? ()
#19 0xbff6947ae0000000 in ?? ()
#20 0x3fbe851ec0000000 in ?? ()
#21 0xbfa570a3e0000000 in ?? ()
#22 0x3f9ee66660000000 in ?? ()
#23 0xc00335c280000000 in ?? ()
#24 0xbfbaa8f5c0000000 in ?? ()
#25 0x3fc23851e0000000 in ?? ()
#26 0xbfe9d1eb80000000 in ?? ()
#27 0xbfcc70a3e0000000 in ?? ()
#28 0xbf911c2900000000 in ?? ()
#29 0x3fa4b33340000000 in ?? ()
#30 0x3fa49eb860000000 in ?? ()
#31 0x3f80451ec0000000 in ?? ()
#32 0xbfa1a66660000000 in ?? ()
#33 0x3faac28f60000000 in ?? ()
#34 0x3fb4e147a0000000 in ?? ()
#35 0x3f98947ae0000000 in ?? ()
#36 0x3faf70a3e0000000 in ?? ()
#37 0x3fab800000000000 in ?? ()
#38 0xbf99dc2900000000 in ?? ()
#39 0xbfbda3d700000000 in ?? ()
#40 0x3fc1b0a3e0000000 in ?? ()
#41 0xbfcfdc2900000000 in ?? ()
#42 0xbfa26e1480000000 in ?? ()
#43 0xbf99d1eb80000000 in ?? ()
#44 0xbfab666660000000 in ?? ()
#45 0x3fc0ca3d80000000 in ?? ()
#46 0xbfcc147ae0000000 in ?? ()
#47 0xbfa22e1480000000 in ?? ()
#48 0x3fbeeb8520000000 in ?? ()
#49 0x3f8d5c2900000000 in ?? ()
#50 0xbfcb3d70a0000000 in ?? ()
#51 0x00007ffd2913ea00 in ?? ()
#52 0x0000000002d085a0 in ?? ()
#53 0x00007ffd2913e6a0 in ?? ()
#54 0x00002afebcac66bf in cling::Interpreter::EvaluateInternal(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, cling::CompilationOptions, cling::Value*, cling::Transaction**, unsigned long) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#55 0x00002afebcac6c2f in cling::Interpreter::process(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, cling::Value*, cling::Transaction**, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#56 0x00002afebcb71b2d in cling::MetaProcessor::process(llvm::StringRef, cling::Interpreter::CompilationResult&, cling::Value*, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#57 0x00002afebca24b3e in HandleInterpreterException(cling::MetaProcessor*, char const*, cling::Interpreter::CompilationResult&, cling::Value*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#58 0x00002afebca33eec in TCling::ProcessLine(char const*, TInterpreter::EErrorCode*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#59 0x00002afeba803a4d in TApplication::ProcessLine(char const*, bool, int*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#60 0x00002afeba418782 in TRint::ProcessLineNr(char const*, char const*, int*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libRint.so
#61 0x00002afeba418ad4 in TRint::HandleTermInput() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libRint.so
#62 0x00002afeba94d2b8 in TUnixSystem::CheckDescriptors() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#63 0x00002afeba94ea98 in TUnixSystem::DispatchOneEvent(bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#64 0x00002afeba86b811 in TSystem::Run() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#65 0x00002afeba801aff in TApplication::Run(bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#66 0x00002afeba41a0fc in TRint::Run(bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libRint.so
#67 0x000000000040097a in main ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x00002afec7b63e5c in ?? ()
#6  0x3f915eb860000000 in ?? ()
#7  0xbf807851e0000000 in ?? ()
#8  0xbfce570a40000000 in ?? ()
#9  0x3fc18cccc0000000 in ?? ()
#10 0xbfe0b851e0000000 in ?? ()
#11 0xc006b33340000000 in ?? ()
#12 0xc01651eb80000000 in ?? ()
#13 0x3fb7a3d700000000 in ?? ()
#14 0xc034f0a3e0000000 in ?? ()
#15 0xbfdb75c280000000 in ?? ()
#16 0x3fb7b851e0000000 in ?? ()
#17 0x3f5775c280000000 in ?? ()
#18 0xc01d70a3e0000000 in ?? ()
#19 0xbff6947ae0000000 in ?? ()
#20 0x3fbe851ec0000000 in ?? ()
#21 0xbfa570a3e0000000 in ?? ()
#22 0x3f9ee66660000000 in ?? ()
#23 0xc00335c280000000 in ?? ()
#24 0xbfbaa8f5c0000000 in ?? ()
#25 0x3fc23851e0000000 in ?? ()
#26 0xbfe9d1eb80000000 in ?? ()
#27 0xbfcc70a3e0000000 in ?? ()
#28 0xbf911c2900000000 in ?? ()
#29 0x3fa4b33340000000 in ?? ()
#30 0x3fa49eb860000000 in ?? ()
#31 0x3f80451ec0000000 in ?? ()
#32 0xbfa1a66660000000 in ?? ()
#33 0x3faac28f60000000 in ?? ()
#34 0x3fb4e147a0000000 in ?? ()
#35 0x3f98947ae0000000 in ?? ()
#36 0x3faf70a3e0000000 in ?? ()
#37 0x3fab800000000000 in ?? ()
#38 0xbf99dc2900000000 in ?? ()
#39 0xbfbda3d700000000 in ?? ()
#40 0x3fc1b0a3e0000000 in ?? ()
#41 0xbfcfdc2900000000 in ?? ()
#42 0xbfa26e1480000000 in ?? ()
#43 0xbf99d1eb80000000 in ?? ()
#44 0xbfab666660000000 in ?? ()
#45 0x3fc0ca3d80000000 in ?? ()
#46 0xbfcc147ae0000000 in ?? ()
#47 0xbfa22e1480000000 in ?? ()
#48 0x3fbeeb8520000000 in ?? ()
#49 0x3f8d5c2900000000 in ?? ()
#50 0xbfcb3d70a0000000 in ?? ()
#51 0x00007ffd2913ea00 in ?? ()
#52 0x0000000002d085a0 in ?? ()
#53 0x00007ffd2913e6a0 in ?? ()
#54 0x00002afebcac66bf in cling::Interpreter::EvaluateInternal(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, cling::CompilationOptions, cling::Value*, cling::Transaction**, unsigned long) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#55 0x00002afebcac6c2f in cling::Interpreter::process(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, cling::Value*, cling::Transaction**, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#56 0x00002afebcb71b2d in cling::MetaProcessor::process(llvm::StringRef, cling::Interpreter::CompilationResult&, cling::Value*, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
#57 0x00002afebca24b3e in HandleInterpreterException(cling::MetaProcessor*, char const*, cling::Interpreter::CompilationResult&, cling::Value*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCling.so
===========================================================


