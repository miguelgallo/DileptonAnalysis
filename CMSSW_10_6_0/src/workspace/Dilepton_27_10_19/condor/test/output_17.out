copying file
running
0cortes: 3292867
1cortes: 48700
2cortes: 45274
3cortes: 43466
4cortes: 20258
5cortes: 20258
6cortes: 11605
7cortes: 1629
8cortes: 18
moving file
